readthedocs/rtd_tests/tests/test_config_integration.py

```diff
@@ -81,30 +82,35 @@ def test_python_supported_versions_default_image_1_0(self, load_config):
         self.project.enable_pdf_build = True
         self.project.save()
         config = load_yaml_config(self.version)
-        self.assertEqual(load_config.call_count, 1)
-        load_config.assert_has_calls([
-            mock.call(
+
+        expected_env_config = {
+            'allow_v2': mock.ANY,
+            'build': {'image': 'readthedocs/build:1.0'},
+            'name': mock.ANY,
```

Comment:
I forgot to remove this in the merge (I guess travis is going to fail)

```suggestion
```

Reply:
Yes, it failed for this reason only.
I have commited the suggested changes.